Activities/DocAcquire/DocAcquire.Activities/UploadDocument.cs

```diff
@@ -0,0 +1,65 @@
+﻿using DocAcquire.Activities;
+using DocAcquire.Activities.Properties;
+using DocAcquire.DataContracts;
+using System;
+using System.Activities;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace DocAcquire.Activities
+{
+    public class UploadDocument : AsyncTaskCodeActivity
+    {
+        private readonly IDocumentUploadService documentService;
+
+        public UploadDocument(IDocumentUploadService documentService)
```

Comment:
public types get exposed inside the workfow for people to use. As IDocumentUploadService seems to be an internal concern for code separation make it internal as well as this constructor. For unit tests it can be made accessible using InternalsVisibleTo

Reply:
done.